### PR TITLE
Have one documentation page about Netdata Charts

### PR DESCRIPTION
**Pull request by Ancairon**
##### Summary

Along with @sashwathn we discussed about having one page talking about charts.

So this PR includes:

- put info from `docs/dashboard/dimensions-contexts-families.md` to `docs/cloud/visualize/interact-new-charts.md`

And while I was fixing the referring links to that file, I discovered that there exists this https://github.com/netdata/netdata/blob/master/docs/dashboard/customize.md, which looks outdated, and seems like was left out of the "One dashboard everywhere" project, so I removed that.

Then there was a link to that file in https://github.com/netdata/netdata/blob/master/web/server/README.md, and I think the best thing to do is to remove the link for now.

I don't know who should take a look at the file, but I don't see that these files got touched with the recent project of removing the old agent dashboard (and maybe they should have?).

Let's discuss what to do with these files.

### CODE REVIEW

**Comment by hugovalente-pm** [comment]
> And while I was fixing the referring links to that file, I discovered that there exists this https://github.com/netdata/netdata/blob/master/docs/dashboard/customize.md, which looks outdated, and seems like was left out of the "One dashboard everywhere" project, so I removed that.

I don't think we should remove this since it is there still as part of the `/v1` dashboard, e.g. http://localhost:19999/v1. We should put some kind of disclaimer saying that his is only applicable to that version of the dashboard and doesn't affect the [Netdata Dashboard](https://learn.netdata.cloud/docs/visualizations/accessing-netdata-dashboards) 

> Then there was a link to that file in https://github.com/netdata/netdata/blob/master/web/server/README.md, and I think the best thing to do is to remove the link for now.

I'm not sure of the technicalities on this but considering the above, we still having `/v1`, and the Agent still needs to provide a webserver to provide the new dashboard we shouldn't remove it, it will probably need a review but not sure the best person to check it, maybe @novykh ?

I know @underhood was involved on discussions and maybe a POC on new webserver [H2O](https://github.com/netdata/netdata/issues/15663) but don't think this is there yet.

**Comment by Ancairon** [comment]
@hugovalente-pm I modified the PR, not removing the file but adding a disclaimer.

My guess is that custom.md should not be on Learn, but exist on github, right?

**Comment by hugovalente-pm** [comment]
> My guess is that custom.md should not be on Learn, but exist on github, right?

can be on learn since, for example, we also left this page there https://learn.netdata.cloud/docs/visualizations/legacy-agent-dashboard

**Review by hugovalente-pm — DISMISSED** [review]

File: docs/cloud/visualize/interact-new-charts.md [hunk 1]
```diff
@@ -37,6 +37,65 @@ With a quick glance you have immediate information available at your disposal:
...
 
+## Fundemental elements
+
+While Netdata's charts require no configuration and are easy to interact with, they have a lot of underlying complexity. To meaningfully organize charts out of the box based on what's happening in your nodes, Netdata uses the concepts of [dimensions](#dimensions), [contexts](#contexts), and [families](#families).
+
+Understanding how these work will help you more easily navigate the dashboard,
+[write new alerts](https://github.com/netdata/netdata/blob/master/health/REFERENCE.md), or play around
+with the [API](https://github.com/netdata/netdata/blob/master/web/api/README.md).
+
+### Dimensions
```
Comment (MrZammler) [inline docs/cloud/visualize/interact-new-charts.md:48]: Should this be 2 `#`, or the other sections 3 `#`?

Comment (Ancairon) [reply docs/cloud/visualize/interact-new-charts.md:48]: the latter, fixing, nice catch

File: docs/cloud/visualize/interact-new-charts.md [hunk 2]
```diff
@@ -37,6 +37,65 @@ With a quick glance you have immediate information available at your disposal:
...
+Beneath every chart (or on the right-side if you configure the dashboard) is a legend of dimensions. When there are
+multiple dimensions, you'll see a different entry in the legend for each dimension.
+
+The **Apps CPU Time** chart (with the [context](#contexts) `apps.cpu`), which visualizes CPU utilization of
+different types of processes/services/applications on your node, always provides a vibrant example of a chart with
+multiple dimensions.
+
+Dimensions can be [hidden](#show-and-hide-dimensions) to help you focus your attention.
+
+## Contexts
```
Comment (Ancairon) [inline docs/cloud/visualize/interact-new-charts.md:64]:
```suggestion
### Contexts
```

File: docs/cloud/visualize/interact-new-charts.md [hunk 3]
```diff
@@ -37,6 +37,65 @@ With a quick glance you have immediate information available at your disposal:
...
+
+For example, the **Apps CPU Time** has the context `apps.cpu`. A little further down on the dashboard is a similar
+chart, **Apps Real Memory (w/o shared)** with the context `apps.mem`. The `apps` portion of the context is the **type**,
+whereas anything after the `.` is specified either by the chart's developer or by the [family](#families).
+
+By default, a chart's type affects where it fits in the menu, while its family creates submenus.
+
+Netdata also relies on contexts for [alert configuration](https://github.com/netdata/netdata/blob/master/health/REFERENCE.md) (the [`on` line](https://github.com/netdata/netdata/blob/master/health/REFERENCE.md#alert-line-on)).
+
+## Families
```
Comment (Ancairon) [inline docs/cloud/visualize/interact-new-charts.md:76]:
```suggestion
### Families
```

**Review by MrZammler — APPROVED** [review]

**Review by Dim-P — APPROVED** [review]

**Pull request merged**